src/sections/offering/get-insured.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react'
+import { Container, InnerContainer } from './insured-amount'
+import { GetInsuredButton, LinkTag } from './price-info'
```

Comment:
I think we should move components that we reuse to some place where it's clear they're reused, so we don't get cross-component (aka silent) dependencies